### ISSUE TITLE
Show track info in left-hand panel in player (optional)

### ISSUE THREAD

**Issue by pccasto** [pccasto/moode#11]
would be nice / useful to see the track number on the left hand side - either in the track title line, or after it in the Artist - Album line.

**Comment by moodeaudio** [comment]
You mean in the main Playlist?

The tracks are added to Playlist in Track tag order but currently the Track# only shows up under the cover art when a track is playing.

I'm sure it wouldn't be hard to add to the Playlist item line but where to put it that wouldn't make the Playlist look cluttered. It already has the Playlist item# on the far left.

This is where a visual mockup would help.

**Comment by pccasto** [comment]
Yes - now that 5.0 is out, I'll see if I can do a mock-up that makes sense.
